package com.aditya.jaca2;

public class JavaExceptionalHandling {
	public static void main(String[] args) {
		System.out.println("Exceptional Handling");
	}
}

/*class ArithmeticException {
	public static void main(String[] args) {
		try {
			int data = 100 / 0;
		} catch (ArithmeticException e) {
			System.out.println(e);
			System.out.println("rest of the code.");
		}
	}
}

class JavaExceptionExample {
	public static void main(String args[]) {
		try {
			// code that may raise exception
			int data = 100 / 0;
		} catch (ArithmeticException e) {
			System.out.println(e);
		}
		// rest code of the program
		System.out.println("rest of the code...");
	}
}

class TryCatchExample2 {  
	  
    public static void main(String[] args) {  
        try  
        {  
        int data=50/0; //may throw exception   
        }  
            //handling the exception  
        catch(ArithmeticException e)  
        {  
            System.out.println(e);  
        }  
        System.out.println("rest of the code");  
    }  
      
}  */